assets/src/modules/LizmapMap.js

```diff
@@ -38,11 +40,29 @@ export default class LizmapMap {
 
     }
 
-    get zoom () {
+    get zoom() {
         return this._zoom;
     }
 
-    get baseLayerGroup () {
+    set zoom(zoom) {
+        this._zoom = zoom;
```

Comment:
To avoid potential infinite loop, you should add  `if(this._zoom == zoom) { return;}` before the assignation.